File: Arcade/Intro/Java/AlphabeticShift.java
import java.util.stream.Collectors;

final class AlphabeticShift {

	String alphabeticShift(String inputString) {
	    return inputString.chars().boxed()
	        .map(c -> c == 122 ? 'a' : (char)(c + 1))
	        .map(Object::toString)
	        .collect(Collectors.joining());
	}
}

File: Arcade/Intro/javascript/alphabeticShift.js
function alphabeticShift(inputString) {
    let chars = [];
    for(let c of inputString) {
        chars.push(c === 'z' 
            ? 'a'
            : String.fromCharCode((c.charCodeAt(0) + 1)));
    }
    return chars.join("");
}